client/src/components/Console.tsx

```diff
@@ -97,6 +97,14 @@ export default function Console() {
     }
   }, [selectedView, refreshNeeded]);
 
+  const [showSideBar, setShowSideBar] = useState(true);
```

Comment:
Perhaps a little margin on the left side can be better

Reply:
added a bit when closed